lua/hbac/telescope/init.lua

```diff
@@ -0,0 +1,100 @@
+if not pcall(require, "telescope") then
+	return false
+end
```

Comment:
Let's be explicit?

Reply:
https://github.com/al-ce/hbac.nvim/commit/4c01fd575ea37d3c79fe318af37f5ba9031aabcc

How does this look @ditsuke? I moved the notification out of the function in `hbac.command`, it was going to get a bit messy.

Reply:
Looks good! Small nit, since the Telescope integration is optional it might be better to check those dependencies only if that component is invoked.

Reply:
Thanks! I thought that was the case already, since as far as I can tell the telescope module is only required if a user executes `:Hbac telescope` or `require("hbac").telescope()`, but I may have missed some other spot?

Reply:
As far as I can see the telescope module is only required during the execution of telescope command, so it should be OK.

Furthermore the module itself should not be required unless the dependencies are set. So I think it is apropriate to check at the top of the module.

Reply:
My bad, looks good then 💯